src/screens/IndexScreen.js

```diff
@@ -390,6 +397,21 @@ export const IndexScreen = ({ navigation, route }) => {
                 <ListComponent
                   ListHeaderComponent={
                     <>
+                      {query === QUERY_TYPES.EVENT_RECORDS && !!eventListIntro && !showCalendar && (
+                        <>
+                          <Wrapper>
+                            <RegularText small>{eventListIntro.introText}</RegularText>
+                          </Wrapper>
```

Comment:
check if `!!eventListIntro.introText`? what if there is noch introtext but only a button? i think we should not render the first wrapper in here.

Reply:
i meant to check for presence around the whole `<Wrapper>...</Wrapper>`

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/c5eb0ce08630b8f7884465ae161431309b8946ef